src/twisted/application/runner/_pidfile.py

```diff
@@ -0,0 +1,303 @@
+# -*- test-case-name: twisted.application.runner.test.test_pidfile -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+PID file.
+"""
```

Comment:
Hrm, I've never used that…

It doesn't seem appropriate for private API if the point is to let developers know how far back they can go and use the API.  `twisted.application.runner` is still undergoing a fair bit of incompatible churn (this is part of that churn), so I think we should hold off until the API ius stable enough to make public.